calblueprint/ogsc
# Editing name and role for user profile

# Editing name and role for user profile

lets admins edit name and role of a user & lets non-admin users edit their own name

NOTE: also fixes a pagination bug on the players table (was slicing wrong before)

## TODO (if applicable)

N/A

## How to Test/Use PR feature

log in as an admin & check if you can edit any user's name & role
log in as any other user and make sure you can edit only your name

## PR Checklist

Does your branch (check if true):


## Related PRs

N/A

## Migrations

N/A

## Screenshots
non-admin users view of their own profile
![image](https://user-images.githubusercontent.com/65790318/116969613-6ae80200-ac6b-11eb-9341-e207879afd7f.png)

admin users view of any profile
![image](https://user-images.githubusercontent.com/65790318/116969718-9539bf80-ac6b-11eb-8bfc-ff897bb9c6b9.png)

_Required for visual changes_

CC: @sydbui
